content/docs/introduction/glossary.md

```diff
@@ -35,17 +35,16 @@ a single metric if it is part of direct instrumentation, or many metrics if it i
 
 ### Direct instrumentation
 
-Direct instrumentation is instrumentation added inline as part of the source code
-of a program.
+Direct instrumentation is instrumentation added inline as part of the source code of a program, using a [client library](#client-library).
 
 ### Endpoint
 
 A source of metrics that can be scraped, usually corresponding to a single process.
 
 ### Exporter
 
-An exporter is a binary that exposes Prometheus metrics, commonly by converting
-metrics that are exposed in a non-Prometheus format into a format Prometheus supports.
+An exporter is a binary running alongside the application you
+want to obtain metrics from, that exposes Prometheus metrics, commonly by converting metrics that are exposed in a non-Prometheus format into a format Prometheus supports.
```

Comment:
This is getting a bit hard to parse. I'd split it into two sentences:
```
An exporter is a binary running alongside the application you
want to obtain metrics from. The exporter exposes Prometheus metrics, commonly by converting metrics that are exposed in a non-Prometheus format into a format that Prometheus supports.
```

Reply:
Looks good to me

Reply:
@avivace if you could update your PR with the suggested change, we could merge it.

Reply:
@beorn7 Yup, sorry, got busy lately. Will work on it as soon as I have some time